models/iaf_psc_exp_multisynapse.cpp

```diff
@@ -344,10 +344,10 @@ iaf_psc_exp_multisynapse::update( const Time& origin, const long from, const lon
   }
 }
 
-port
-iaf_psc_exp_multisynapse::handles_test_event( SpikeEvent&, rport receptor_type )
+size_t
+iaf_psc_exp_multisynapse::handles_test_event( SpikeEvent&, size_t receptor_type )
 {
-  if ( receptor_type <= 0 or receptor_type > static_cast< port >( P_.n_receptors_() ) )
+  if ( receptor_type <= 0 or receptor_type > static_cast< size_t >( P_.n_receptors_() ) )
```

Comment:
See above, and also change everywhere else, where `P_.n_receptors_()` is used in a similar way.
```suggestion
  if ( receptor_type <= 0 or receptor_type > P_.n_receptors_() )
```